packages/squiggle-lang/src/value/index.ts

```diff
@@ -112,7 +112,10 @@ class VArray extends BaseValue implements Indexable {
     }
 
     for (let i = 0; i < this.value.length; i++) {
-      isEqual(this.value[i], other.value[i]);
+      const _isEqual = isEqual(this.value[i], other.value[i]);
```

Comment:
I got curious about this...

[Result value must be used check](https://github.com/microsoft/TypeScript/issues/8240) issue on TS repo was opened 8 years ago (latest idea there is `nodiscard` keyword), [eslint issue](https://github.com/eslint/eslint/issues/12980) wasn't accepted either (and it would be hard to do without extra syntax).

Oh well.

Reply:
Huh, that's interesting, good find

Reply:
Seems like a bug that should be easy for an LLM to detect at least.